researchseminarsdotorg.py

```diff
@@ -0,0 +1,103 @@
+import os
+from requests import get, post
+
+SPEAKERS_CORNER_SEMINAR_SERIES = {"series_id": "speakerscorner",
+           "name": "Speakers\' Corner",
+           "is_conference": False,
+           "topics": [""], # TODO: Get a list of topics
+           "language": "en",
+           "institutions": ["Global"],
+           "timezone": "America/New_York",
+           "homepage": "https://virtualscienceforum.org/speakerscorner.md",
+           "visibility": 1, # 0=private, 1=unlisted, 2=public
+           "access_control": 0, # 0=open, see schema for more
+           "slots": [""],
+           "organizers": [{"name": "Virtual Science Forum",
```

Comment:
Note: Update timezone & Is this the right account?

Reply:
That's just the contact information on the confirmation email AFAIK, so doesn't matter. I don't think we have any knowledge of the timezone, so UTC is fine.

Reply:
Ok, I believe I have already changed that in the last push. Will make sure though